lib/src/controls/navigation/navigation_view/body.dart

```diff
@@ -140,27 +140,39 @@ class _NavigationBodyState extends State<_NavigationBody> {
             child: child,
           );
         },
-        child: KeyedSubtree(
-          key: widget.itemKey,
-          child: PageView.builder(
-            key: _pageKey,
-            physics: const NeverScrollableScrollPhysics(),
-            controller: pageController,
-            itemCount: view.pane!.effectiveItems.length,
-            itemBuilder: (context, index) {
-              final isSelected = view.pane!.selected == index;
-              final item = view.pane!.effectiveItems[index];
-
-              return ExcludeFocus(
-                key: item.bodyKey,
-                excluding: !isSelected,
-                child: FocusTraversalGroup(
-                  child: widget.paneBodyBuilder?.call(item.body) ?? item.body,
-                ),
-              );
-            },
-          ),
-        ),
+        child: () {
+          final paneBodyBuilder = widget.paneBodyBuilder;
+          if (paneBodyBuilder != null) {
+            return FocusTraversalGroup(
+              key: widget.itemKey,
+              child: paneBodyBuilder.call(view.pane?.selected != null
+                  ? view.pane?.selectedItem.body
+                  : null),
+            );
```

Comment:
`itemKey` should be for every item. What do you think?

Reply:
The code path where `paneBodyBuilder` is non-null essentially reverts to the [old behavior](https://github.com/bdlukaa/fluent_ui/pull/607/files#diff-b33184314dc53c5740bf048c731cfa0f34c0d6e05b02311863562e81ef7d5387R137). It looks like the itemKey being passed in to `_NavigatorBody` has the value of ```itemKey: ValueKey(pane.selected ?? -1),```, so this would be having the `FocusTraversalGroup` be differentiated for each group of controls separately for each pane item (what we want)?

I tried removing the `key` from the `FocusTraversalGroup` entirely and it didn't seem to break anything, so I'm not sure why it's even needed at all here.